scripts/utils/checkProposal.ts

```diff
@@ -45,12 +48,12 @@ async function checkProposal() {
     console.log('Vote already began');
   }
 
-  try {
-    await governorAlpha.castVote(proposalNo, true, { from: voterAddress });
-    console.log('Casted vote');
-  } catch {
-    console.log('Already voted');
-  }
+  /*try {*/
+    await governorAlpha.connect(voterSigner).castVote(proposalNo, true);
+    console.log('Casted vote.');
+  /*} catch {
+    console.log('Already voted, ror some terrible error has occured.');
```

Comment:
Clean up these comments? Add a message saying this will revert if proposal queued